src/event/entity/EntityDamageEvent.php

```diff
@@ -63,6 +63,7 @@ class EntityDamageEvent extends EntityEvent implements Cancellable{
 	public const CAUSE_MAGIC = 13;
 	public const CAUSE_CUSTOM = 14;
 	public const CAUSE_STARVATION = 15;
+	public const CAUSE_FALLING_ANVIL = 16;
```

Comment:
FALLING_BLOCK would be better (in case more blocks like this are added in the future).

Reply:
Yeah you bring up a good point I will change this.

Reply:
Dripstone i.e.